mirrord/layer/src/file/filter.rs

```diff
@@ -82,8 +82,9 @@ fn generate_local_set() -> RegexSet {
         // asdf
         r".*/\.tool-versions$",
         // macOS
-        r"^/Users/.*$",
-        r"^/Library/.*$",
+        r"^/Users",
+        r"^/Library",
+        r"^/Applications",
```

Comment:
We could be more precise:
```suggestion
        r"^/Users(/.*)?$",
        r"^/Library(/.*)?$",
        r"^/Applications(/.*)?$",
```

Reply:
What cases would it solve though?

Reply:
It would prevent matching paths that have those directories as prefixes, e.g. `/Users.db` or `/LibraryFiles/whatever` probably not very important.

Reply:
💀